_pages/dev/ssr/server-side-rendering-optimization.md

```diff
@@ -115,21 +116,54 @@ The default value is `3000` milliseconds.
 The `cacheSize` setting can also be used when the `cache` setting is set to `false`. This then limits the number of timed-out renders that are kept in a temporary cache and which are waiting to be served with the next request.
 
-It is recommended that the `cacheSize` should be set according to the server's resources (such as the amount of available RAM). It is recommended that you set the `cacheSize`, regardless of whether the `cache` setting is disabled.
+It is recommended that the `cacheSize` should be set according to the server's resources (such as the amount of available RAM), leaving also some room for the spikes of the memory needed for the rendering of pages by the Angular SSR engine. It is recommended that you set the `cacheSize`, regardless of whether the `cache` setting is disabled.
 
 The default `cacheSize` is set to `3000` entries. Before version 2211.19 of Spartacus, no default value was set, which could result in unlimited cached pages for those pages that fell back to CSR due to timeout. This could potentially lead to a memory leak.
 
-The default value is based on a few known values and a few assumptions. In SAP Commerce Cloud, the minimum pod size is 3 GB. To avoid processes from restarting, as a result of exceeding the default upper limit of 60% for memory usage, a safer, lower limit of 50% is set. Consequently, the usable memory that is available by default is calculated to be 3 GB multiplied by 50%, with a result of 1.5 GB. The next calculation considers a typical HTML page to have a size of approximately 350 KB. However, you may have even larger rendered HTML pages in your project. As a precaution, it is assumed that HTML pages could be up to 150% larger, resulting in a maximum page size of 525 KB. Accordingly, the calculation for the default `cacheSize` is 1.5 GB divided by 525 KB, leading to a result of 3070. This value is rounded down to provide the final `cacheSize` default of `3000` entries.
+~The default value is based on a few known values and a few assumptions. In SAP Commerce Cloud, the minimum pod size is 3 GB. To avoid processes from restarting, as a result of exceeding the default upper limit of 60% for memory usage, a safer, lower limit of 50% is set. Consequently, the usable memory that is available by default is calculated to be 3 GB multiplied by 50%, with a result of 1.5 GB. The next calculation considers a typical HTML page to have a size of approximately 350 KB. However, you may have even larger rendered HTML pages in your project. As a precaution, it is assumed that HTML pages could be up to 150% larger, resulting in a maximum page size of 525 KB. Accordingly, the calculation for the default `cacheSize` is 1.5 GB divided by 525 KB, leading to a result of 3070. This value is rounded down to provide the final `cacheSize` default of `3000` entries.~
+
+UPDATE: The above reasoning was not valid as it didn't leave room for the spikes of the memory needed for the rendering of pages by the Angular SSR engine.
```

Comment:
Notice for the reviewer: I don't know what to do with the wrong calculation for the old property:
a) should we cross it out (for visibility) and add a disclaimer that it was wrong
b) OR we should remove it
?

Reply:
I tried to add some text that would have allowed us to keep this paragraph, by explaining that the calculation was no longer valid (but remains as a record of what we had said)... But because the text goes into such a lot of detail about how the calculation is made -- and it happens to be wrong -- and there is a similar but different description of a calculation in the `cacheSizeMemory` section, I am deciding to delete this paragraph after all because, experience tells me that people do not read very carefully, and leaving it in may lead to complaints about how we are contradicting ourselves with these different calculations in different sections, and that it is confusing, etc etc... 🤷 😊